### PR TITLE
Update Hy support by adding a reference to an Atom grammar.

### PR DESCRIPTION
**Pull request by rwtolbert**

### CODE REVIEW

**Comment by rwtolbert** [comment]
Is there more I need to do here?

Hy has been supported for a while in Linguist and repos are marked correctly. This is just an attempt to get syntax highlighting back due to the change in how that works.

I used Atom as the simplest path, copying from the Clojure Atom grammar.

Do I need to do TextMate instead?

Or is there some other piece I'm missing?

**Comment by pchaigno** [comment]
@rwtolbert I'm not from GitHub but I may be able to help you.
You will need to update the `tm_scope` to `source.hy` in `languages.yml`.
If you can also update your branch against master it would probably help ;)

**Comment by rwtolbert** [comment]
Thanks for the hints. I'll get those changes pushed.

**Comment by rwtolbert** [comment]
and here are a [couple](https://lightshow.githubapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_url=https%3A%2F%2Fgithub.com%2Frwtolbert%2Flanguage-hy%2Fblob%2Fmaster%2Fgrammars%2Fhy.cson&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Falgernon%2Fadderall%2Fblob%2Fmaster%2Fadderall%2Fdsl.hy&code=) of [examples](https://lightshow.githubapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_url=https%3A%2F%2Fgithub.com%2Frwtolbert%2Flanguage-hy%2Fblob%2Fmaster%2Fgrammars%2Fhy.cson&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Fhylang%2Fhy%2Fblob%2Fmaster%2Fhy%2Fcore%2Flanguage.hy&code=) in use.

**Comment by pchaigno** [comment]
Looks great!
I guess you only have to wait for someone from GitHub to review and merge now :-)

(For reviewer, [license available here](https://github.com/rwtolbert/language-hy/blob/master/LICENSE.md).)

**Comment by rwtolbert** [comment]
Thanks again for the help.

File: lib/linguist/languages.yml [hunk 1]
```diff
@@ -1267,13 +1267,13 @@ Haxe:
 
 Hy:
   type: programming
-  ace_mode: clojure
+  ace_mode: text
```
Comment (aroben) [inline lib/linguist/languages.yml]: What's the reason for this change? Was the clojure Ace mode not working well?

**Comment by aroben** [comment]
@rwtolbert I have a small question above, but overall this looks good. I'll take care of getting it updated/merged once we're clear on the Ace mode issue. Thanks!

**Comment by rwtolbert** [comment]
Hy started as a close match to Clojure but they have started to diverge, so I thought that unless there was a proper Ace mode for Hy, then maybe falling back to "text" made sense.

**Comment by aroben** [comment]
I merged this over in #1974. Thanks!

**Comment by rwtolbert** [comment]
It was old and way out of date. Sorry if I broke anything, but it is better off gone than broken.

Hopefully someone that knows more about Atom and Hy can make a version that works.

**Comment by pchaigno** [comment]
We didn't get any report of broken highlighting, but it's your call.

#3411 will remove the grammar so Hy won't have any syntax highlighting anymore. If you know an alternative to your grammar, we'd love a pull request!
